src/mcpbr/harness.py

```
@@ -1006,6 +1006,99 @@ def _do_upload() -> None:
+    try:
+        from .notifications import NotificationEvent, dispatch_notification
+
+        event = NotificationEvent(
+            event_type=event_type,
+            benchmark=config.benchmark,
+            model=config.model,
+            extra=extra or {},
+        )
+        dispatch_notification(notify_config, event)
+    except Exception:
+        pass  # Notifications must never block the evaluation flow
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)